cmake/Modules/LocalRav1e.cmake

```diff
@@ -1,5 +1,5 @@
 set(AVIF_LOCAL_RAV1E_GIT_TAG v0.7.1)
-set(AVIF_LOCAL_CORROSION_GIT_TAG v0.4.4)
+set(AVIF_LOCAL_CORROSION_GIT_TAG v0.4.10)
```

Comment:
Note that I know Corrosion v0.5.0 is available. The reason I did not upgrade to v0.5.0 in this pull request is that I didn't spend the time understanding the breaking change mentioned in the release notes. We can upgrade to v0.5.0 in a follow-up pull request.